Jenkinsfile

```diff
@@ -186,6 +186,12 @@ ansiColor('xterm') {
             }
           }
 
+          stage('Check for ES5 Compatibility') {
+            sh'''
```

Comment:
Could you add this to the end of the `Build for CDN` stage instead since it is checking the files built in that stage.

Reply:
I thought i did.

Reply:
Instead of creating a new stage, this verification step should be in the `Build for CDN` stage instead.

Reply:
ohhhh.. I got you. I misread the comment AGAIN.